atmel-samd/common-hal/audiobusio/PDMIn.c

```diff
@@ -213,74 +231,99 @@ static void setup_dma(audiobusio_pdmin_obj_t* self, uint32_t length,
 void start_dma(audiobusio_pdmin_obj_t* self) {
     dma_start_transfer_job(&audio_dma);
     tc_start_counter(MP_STATE_VM(audiodma_block_counter));
-    i2s_clock_unit_enable(&self->i2s_instance, self->clock_unit);
-    i2s_serializer_enable(&self->i2s_instance, self->serializer);
     I2S->DATA[1].reg = I2S->DATA[1].reg;
 }
 
 void stop_dma(audiobusio_pdmin_obj_t* self) {
-    // Turn off the I2S clock and serializer. Peripheral is still enabled.
-    i2s_serializer_disable(&self->i2s_instance, self->serializer);
-    i2s_clock_unit_disable(&self->i2s_instance, self->clock_unit);
-
-    // Shutdown the DMA
+    // Shutdown the DMA: serializer keeps running.
     tc_stop_counter(MP_STATE_VM(audiodma_block_counter));
     dma_abort_job(&audio_dma);
 }
 
-static const uint16_t sinc_filter[64] = {
-    0, 1, 6, 16, 29, 49, 75, 108,
-    149, 200, 261, 334, 418, 514, 622, 742,
-    872, 1012, 1161, 1315, 1472, 1631, 1787, 1938,
-    2081, 2212, 2329, 2429, 2509, 2568, 2604, 2616,
-    2604, 2568, 2509, 2429, 2329, 2212, 2081, 1938,
-    1787, 1631, 1472, 1315, 1161, 1012, 872, 742,
-    622, 514, 418, 334, 261, 200, 149, 108,
-    75, 49, 29, 16, 6, 1, 0, 0
+/* // a windowed sinc filter for 16 kHz, 64 samples. */
+/* // truncated integer values */
+/* static const uint16_t sinc_filter[OVERSAMPLING] = { */
+/*     0, 1, 6, 16, 29, 49, 75, 108, */
+/*     149, 200, 261, 334, 418, 514, 622, 742, */
+/*     872, 1012, 1161, 1315, 1472, 1631, 1787, 1938, */
+/*     2081, 2212, 2329, 2429, 2509, 2568, 2604, 2616, */
+/*     2604, 2568, 2509, 2429, 2329, 2212, 2081, 1938, */
+/*     1787, 1631, 1472, 1315, 1161, 1012, 872, 742, */
+/*     622, 514, 418, 334, 261, 200, 149, 108, */
+/*     75, 49, 29, 16, 6, 1, 0, 0 */
+/* }; */
+
+/* // a windowed sinc filter for 16 kHz, 64 samples. */
+/* // rounded integer values */
+/* static const uint16_t sinc_filter[OVERSAMPLING] = { */
+/*     0, 2, 7, 16, 30, 49, 75, 108, */
+/*     150, 201, 262, 334, 418, 514, 622, 742, */
+/*     873, 1013, 1161, 1315, 1473, 1631, 1788, 1939, */
+/*     2082, 2213, 2330, 2430, 2510, 2568, 2604, 2616, */
+/*     2604, 2568, 2510, 2430, 2330, 2213, 2082, 1939, */
+/*     1788, 1631, 1473, 1315, 1161, 1013, 873, 742, */
+/*     622, 514, 418, 334, 262, 201, 150, 108, */
+/*     75, 49, 30, 16, 7, 2, 0, 0 */
+/* }; */
```

Comment:
Why are these still in here?

Reply:
Wanted to stash them somewhere, but I can save them elsewhere.